### PR TITLE
Minor editorial changes

### PR DESCRIPTION
**Pull request by chrisn**
Having just reviewed the Remote Playback API spec, I have a few very minor editorial suggestions to hopefully improve the spec.

***
<a href="https://pr-preview.s3.amazonaws.com/chrisn/remote-playback/pull/121.html" title="Last updated on May 30, 2018, 6:50 PM GMT (f6aeecb)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/remote-playback/121/e6a2c27...chrisn:f6aeecb.html" title="Last updated on May 30, 2018, 6:50 PM GMT (f6aeecb)">Diff</a>

### CODE REVIEW

**Comment by markafoltz** [comment]
@chrisn Can you link your w3c and Github accounts to pass the IPR check?

File: index.html [hunk 1]
```diff
@@ -1391,10 +1391,10 @@ <h4>
           <div class="note">
             The remote playback device might not actually stop playback of the
             media when requested by the user agent since it's implementation
```
Comment (markafoltz) [inline index.html]: This still reads awkwardly, suggestion:

> when requested by the user agent; it depends on the implementation of the user agent and the remote playback device.

**Comment by markafoltz** [comment]
Thanks for the PR, made one optional suggestion.

**Comment by chrisn** [comment]
Thanks, Mark. I linked my accounts after submitting the PR, and added your suggested wording.

**Comment by markafoltz** [comment]
Thanks, looks good.  Merging.

**Pull request merged**